Dockerfile

```diff
@@ -0,0 +1,75 @@
+################################################################################
+# Base image
+# Based on https://github.com/andrewmclagan/nginx-hhvm-docker
+################################################################################
+
+FROM nginx
+
+################################################################################
+# Add HHVM repo
+################################################################################
+
+RUN apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62
+RUN apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0x5a16e7281be7a449 && \
+    echo deb http://dl.hhvm.com/debian jessie main | tee /etc/apt/sources.list.d/hhvm.list
+
+################################################################################
+# Install supervisor, HHVM & tools
+################################################################################
+
+RUN apt-get install debian-archive-keyring
+RUN apt-get update -o Retries=25
+RUN apt-get install -my -o Retries=25 \
+	hhvm \
+	wget \
+	sqlite3 \
+	supervisor \
+    && apt-get clean
+
+################################################################################
+# Install tools
+################################################################################
+
+RUN cd $HOME && \
+    wget http://getcomposer.org/composer.phar && \
+    chmod +x composer.phar && \
+    mv composer.phar /usr/local/bin/composer && \
+    wget https://phar.phpunit.de/phpunit.phar && \
+    chmod +x phpunit.phar && \
+    mv phpunit.phar /usr/local/bin/phpunit  
+
+RUN composer global require hirak/prestissimo
+
+################################################################################
+# Configuration
+##############################################################################
+
+COPY ./docker/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+
+COPY ./docker/php.ini /etc/hhvm/php.ini
+
+COPY ./docker/nginx.conf /etc/nginx/nginx.conf
+
+COPY ./docker/conf.d/config-1.conf /etc/nginx/conf.d/config-1.conf
+
+COPY ./docker/sites-enabled/default /etc/nginx/sites-enabled/default
+
+
+################################################################################
+# Copy source
+##############################################################################
+
+COPY ./source/ /var/www/
```

Comment:
Is it a convention to move the files in `source`? There is a `.dockerignore` file that you can use to omit things from the build context, and that way we can use `COPY /var/www`.

Side note: should it be `/var/www/html` or `/var/www/squabble`? I believe that's generally the convention but don't know for sure.